### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by theofidry**
Use of syntax highlighting for better doc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,33 +13,35 @@ Authors
...
-        {
-           // ...
-           "require": {
-               // ...
-               "misd/phone-number-bundle": "~1.0"
-           }
-        }
+ 1. Add the PhoneNumberBundle to your dependencies in the `composer.json`:
+
+```json
```
Comment (xabbuh) [inline README.md]: This will look odd since JSON does not know anything about comments.

Comment (theofidry) [inline README.md]: Yeah but people get it it's not that troublesome

Comment (xabbuh) [inline README.md]: What about using code blocks without JSON specific syntax highlighting?

Comment (theofidry) [inline README.md]: Well I find it less readable but that's just my opinion. So I can let the code blocks if you wish :)

Comment (xabbuh) [inline README.md]: Well, I am not the bundle maintainer. :) I find it more readable the way it is now, but the final decision is, of course, up to @thewilkybarkid.

**Comment by theofidry** [comment]
ping

**Comment by shakaran** [comment]
@theofidry I am not the maintainer, but I suggest you rebase the changes and resolve conflicts since it seems that it has conflicts with master

**Comment by rh389** [comment]
Have to close this as it's well out of date and the source repo has disappeared - sorry it never got pulled. If anyone want's to resubmit they're welcome.

**Comment by shakaran** [comment]
@rh389 well, at least the changes could be extracted from this commit https://github.com/misd-service-development/phone-number-bundle/pull/60/commits/a0574bca8fd81904ccbc2a533be17d3dd1f2f6ec which probably could lead to create another PR, would you accept if I mak it based in that commit for example?

**Comment by rh389** [comment]
Go for it.

**Pull request closed without merging**